From f5aa7c78e7a20d91a7fc58873285ce8f4c020802 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Sun, 17 Nov 2013 02:19:39 +0100
Subject: [PATCH 77/95] ARM: dts: imx53: Fix display pinmux for M53EVK

Use the DISP1 pinmux on M53EVK for the LCD, not the LVDS.

Signed-off-by: Marek Vasut <marex@denx.de>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx53-m53evk.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx53-m53evk.dts b/arch/arm/boot/dts/imx53-m53evk.dts
index 912163d..3ca8ab0 100644
--- a/arch/arm/boot/dts/imx53-m53evk.dts
+++ b/arch/arm/boot/dts/imx53-m53evk.dts
@@ -26,7 +26,7 @@
 			crtcs = <&ipu 1>;
 			interface-pix-fmt = "bgr666";
 			pinctrl-names = "default";
-			pinctrl-0 = <&pinctrl_ipu_disp2>;
+			pinctrl-0 = <&pinctrl_ipu_disp1>;
 
 			display-timings {
 				800x480p60 {
@@ -254,8 +254,8 @@
 			fsl,pins = <MX53_I2C3_PINGRP1>;
 		};
 
-		pinctrl_ipu_disp2: ipudisp2grp {
-			fsl,pins = <MX53_IPU_DISP2_PINGRP1>;
+		pinctrl_ipu_disp1: ipudisp1grp {
+			fsl,pins = <MX53_IPU_DISP1_PINGRP1>;
 		};
 
 		pinctrl_nand: nandgrp {
-- 
1.8.4.3

